packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/MediaLibraryUtils.java

```diff
@@ -231,13 +231,13 @@ static String mapOrderDescriptor(List orderDescriptor) throws IllegalArgumentExc
       if (item instanceof String) {
         String key = convertSortByKey((String) item);
         result.add(key + " DESC");
-      } else if (item instanceof Object[]) {
-        Object array[] = (Object[]) item;
-        if (array.length != 2) {
-          throw new IllegalArgumentException("Array sortBy in assetsOptions has invalid layout.");
+      } else if (item instanceof ArrayList) {
+        ArrayList array = (ArrayList) item;
+        if (array.size() != 2) {
+            throw new IllegalArgumentException("Array sortBy in assetsOptions has invalid layout.");
```

Comment:
```suggestion
          throw new IllegalArgumentException("Array sortBy in assetsOptions has invalid layout.");
```
we indent code with 2 spaces, not 4 😉

Reply:
Thanks for the pointers! Updated!